insights/parsers/docker_list.py

```diff
@@ -85,7 +85,7 @@ def parse_content(self, content):
         """
         self.rows = []
         if len(content) < 2:
-            self.no_data = True
+            self.data = None
```

Comment:
Why not raise a `SkipException` directly?

Reply:
@xiangce I did not wanted to change the original implementation, hence did not use `SkipException`. Do you want to use `SkipException`?

Reply:
Generally, it's recommended to keep the original implementation for any update. But in that case, the original code was made a long time ago and at that time `SkipException` is not introduced to the framework yet.

Since you want to do `enhancement` to it, better involve the relatively `new` features which could also bring external benefits.

Reply:
Got your point!